tox.ini

```diff
@@ -53,7 +53,7 @@ commands = pytest -v --tb native --show-capture=no {posargs} {toxinidir}/tests/
 # BUNDLE_TEST_PATH=tests-bundle/1.7 tox -e full_bundle_tests -- --channel=1.7/stable
 commands =
     pytest -vs --tb native -m deploy --model kubeflow --keep-models {env:BUNDLE_TEST_PATH} {posargs}
-    pytest -vs --tb native -m selenium --model kubeflow {env:BUNDLE_TEST_PATH} {posargs}
+    ; pytest -vs --tb native -m selenium --model kubeflow {env:BUNDLE_TEST_PATH} {posargs}
```

Comment:
Please add comment to why we disabling this and if it is remorary. Reference the issue as well.

Reply:
Good note, pushed a comment for this.